operations/configuration_management/ARP_cache_tuning.md

```diff
@@ -0,0 +1,308 @@
+   ```
+
+1. Commit the change and push it back up to the VCS.
+
+   ```bash
+   git add roles/csm.ncn.sysctl/vars/main.yml
+   git commit -m 'Set ARP cache values for all NCNs'
+   git push --set-upstream origin integration-1.26.0
+   ```
+
+#### Configure CFS to run the `csm.ncn.sysctl` role
+
+1. Create a CFS configuration using the committed changes.
+
+   1. Obtain the commit hash and create a configuration template file.
+
+      ```bash
+      COMMIT=$(git rev-parse --verify HEAD)
+      cat << EOF > arp-settings.json
+      {
+        "layers": [
+          {
+             "cloneUrl": "https://api-gw-service-nmn.local/vcs/cray/csm-config-management.git",
+             "commit": "${COMMIT}",
+             "name": "arp-cache-settings",
+             "playbook": "ncn_sysctl.yml"
+          }
+        ]
+      }
+      EOF
+      ```
+
+   1. Create a CFS configuration from the template file.
+
+      ```bash
+      cray cfs configurations update arp-cache-settings --file ./arp-settings.json
+      ```
+
+      Example output:
+
+      ```toml
+      lastUpdated = "2025-01-06T22:39:46Z"
+      name = "arp-cache-settings"
+      [[layers]]
+      cloneUrl = "https://api-gw-service-nmn.local/vcs/cray/csm-config-management.git"
+      commit = "36811473a8b98e88ef8afee1df021d55eac50114"
+      name = "arp-cache-settings"
+      playbook = "ncn_sysctl.yml"
+      ```
+
+1. Create a CFS session to apply the configuration to the nodes.
+
+   ```bash
+   SESSION=arp-cache-settings-$(date +%Y%m%d%H%M%S)
+   cray cfs sessions create --name "${SESSION}" --configuration-name arp-cache-settings
+   ```
+
+   Example output:
+
+   ```toml
+   name = "arp-cache-settings-20250106224045"   
+   
+   [ansible]
+   config = "cfs-default-ansible-cfg"
+   verbosity = 0   
+   
+   [configuration]
+   limit = ""
+   name = "arp-cache-settings"   
+   
+   [status]
+   artifacts = []   
+   
+   [tags]   
+   
+   [target]
+   definition = "dynamic"
+   groups = []
+   image_map = []   
+   
+   [status.session]
+   startTime = "2025-01-06T22:41:04"
+   status = "pending"
+   succeeded = "none"
+   ```
+
+## Verification
+
+1. Check the CFS session completed successfully.
+
+   ```bash
+   cray cfs sessions describe ${SESSION}
+   ```
+
+   Example output:
+
+   ```ini
+   name = "arp-cache-settings-20250106224045"
+
+   [ansible]
+   config = "cfs-default-ansible-cfg"
+   verbosity = 0
+
+   [configuration]
+   limit = ""
+   name = "arp-cache-settings"
+
+   [status]
+   artifacts = []
+
+   [tags]
+
+   [target]
+   definition = "dynamic"
+   groups = []
+   image_map = []
+
+   [status.session]
+   completionTime = "2025-01-06T22:41:14"
+   job = "cfs-e7962be9-1cfa-4604-bb07-d5ae99be5456"
+   startTime = "2025-01-06T22:41:04"
+   status = "complete"
+   succeeded = "true"
+   ```
+
+   The session status should be "complete" and succeeded should be "true". See the [troubleshooting](#troubleshooting) section if that is not the case.
+
+1. Verify ARP settings.
```

Comment:
```suggestion
1. (`ncn-m#`) Verify ARP settings.
```
This is the formatting to denote where the command should be run. Please adjust in other steps as necessary.